examples/picker/app.js

```diff
@@ -0,0 +1,28 @@
+
+var list = document.querySelector('gaia-fast-list');
+
+list.configure({
+  getSectionName(item) { return item.section; }
+});
+
+list.model = getData();
+
+function getData() {
+  var result = [];
+
+  var letters = 'abcdefghijklmnopqrstuvwyz';
+  var length = letters.length;
+
+  for (var i = 0; i < length; i++) {
+    for (var j = 0; j < 20; j++) {
+      result.push({
+        title: `Title ${j}`,
```

Comment:
nit: we should maybe add `letters[i]` somewhere in the title/body otherwise it looks like the content didn't even changed.

I was going to suggest highlighting the section after the jump, but I think it won't be necessary in real apps where the content is clearly different.